hivemind/modules/api/tests/models/test_event.py

```diff
@@ -0,0 +1,47 @@
+"""
+Tests for API Module Signals
+"""
+import datetime
+
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from hivemind.modules.api.models.event import Event
+
+
+class TestEvent(TestCase):
```

Comment:
Please add tests for the Event `__str__` , `get_full_name` and `get_last_name` methods to make sure we get the proper response.